assets/differentialabundance_report.Rmd

```diff
@@ -671,11 +671,36 @@ foo <- lapply(informative_variables[iv_min_group_sizes > 2], function(iv){
     }
   }
 })
-
 ```
 
 ## Differential analysis
 
+```{r, echo=FALSE, results='asis', eval=params$study_type %in% c('rnaseq')}
+# For DESeq2, add some more explanation to the report
+cat(paste0(
+"The `DESeq2 R` package was used for differential analysis. p-values were adjusted with the ",
+params$deseq2_p_adjust_method,
+" method to reduce the number of false positives (",
+params$features_type,
+"s that are not actually differential). ",
+ucfirst(params$features_type),
+"s were considered differential if, for the respective contrast, the adjusted p-value was equal to or lower than ",
+params$deseq2_alpha,
+" and the absolute log2 fold change was equal to or higher than ",
+params$deseq2_lfc_threshold,
+"."
+))
```

Comment:
```suggestion
# For DESeq2, add some more explanation to the report
cat(paste0(
"The `DESeq2 R` package was used for differential analysis. p-values were adjusted with the ", params$deseq2_p_adjust_method, " method to reduce the number of false positives. ", ucfirst(params$features_type), "s were considered differential if, for the respective contrast, the adjusted p-value was equal to or lower than ", params$deseq2_alpha, " and the absolute log2 fold change was equal to or higher than ", params$deseq2_lfc_threshold, "."
))
```

I think we should expect people to understand what a false positive is.

Reply:
I agree! I just took the text that was written by someone else from another pipeline :'D